pygmt/helpers/decorators.py

```diff
@@ -11,6 +11,7 @@
 from inspect import Parameter, signature
 
 import numpy as np
+import pygmt
```

Comment:
I don't like the `import pygmt` statement, but using `from pygmt.figure import Figure` doesn't work.

Reply:
See https://github.com/GenericMappingTools/pygmt/pull/1548#discussion_r719821094 for a workaround.
```suggestion
```